cli.py

```diff
@@ -184,6 +186,17 @@ def log(ctx, selector: str, follow: bool, tail: int or None, grep: str or None,
     handler.log(selector, follow, tail, grep, download)
 
 
+@cli.command()
```

Comment:
As this is yarn specific, this should be nested under the yarn subcommand, see queue for example:
```python
@yarn.command()
@click.pass_context
def queue(ctx):
```

Reply:
fixed.